sdks/java/io/pulsar/src/main/java/org/apache/beam/sdk/io/pulsar/ReadFromPulsarDoFn.java

```diff
@@ -207,7 +189,7 @@ public OffsetRangeTracker restrictionTracker(
     }
 
     PulsarLatestOffsetEstimator offsetEstimator =
-        new PulsarLatestOffsetEstimator(this.admin, pulsarSource.getTopic());
+        new PulsarLatestOffsetEstimator(null /*this.admin*/, pulsarSource.getTopic());
```

Comment:
passing in `null` here will cause an NPE when we try to access the `client` within the `PulsarLatestOffsetEstimator`